expandybird/expander/expander.go

```diff
@@ -73,17 +137,41 @@ func NewTemplateFromFileNames(
 	templateFileName string,
 	importFileNames []string,
 ) (*common.Template, error) {
-	name := path.Base(templateFileName)
 	content, err := ioutil.ReadFile(templateFileName)
 	if err != nil {
-		return nil, fmt.Errorf("cannot read template file (%s): %s", err, templateFileName)
+		return nil, fmt.Errorf("cannot read template file %s: %s", templateFileName, err)
 	}
 
+	name := path.Base(templateFileName)
+	return newTemplateFromContentAndImports(name, string(content), importFileNames)
```

Comment:
I prefer assigning the variable, since it makes the method call less cluttered (no nested parentheses), so if it's not a big deal, let's leave it as is.